apps/ledger-live-desktop/src/renderer/components/Illustration.tsx

```diff
@@ -6,6 +6,7 @@ type Props = {
   lightSource: string;
   darkSource: string;
   size: number;
+  optionalHeight?: number;
```

Comment:
Why did you call it `optional`?

Reply:
Because we do not have a `width` prop. If we only provide a size it will be a square. So I thought adding optional would make sense.